xamarin/Xamarin.Forms
# Force Shell TitleView to height of container and fix flyout header scroll behaviors

### Description of Change ###

Fixed the TitleView on Shell to force its bounds to the size of the titleview container on iOS. It's currently using the measured height of View to layout. Also fixed the scroll behaviors on the shell flyout. The TitleView and the FlyoutHeader both use the same control helpers to renderer so the fix applied both places

### Issues Resolved ### 
- fixes #13476 
- fixes #13511
- fixes #13561 

### Platforms Affected ### 
- iOS


### Testing Procedure ###
- UI Tests included
- Test the Flyout Header and the Shell Title View to verify that they work correctly

### PR Checklist ###
